plugin.video.vtm.go/addon.xml

```diff
@@ -22,8 +22,9 @@
         <disclaimer lang="nl_NL">Deze add-on wordt niet ondersteund door DPG Media en wordt aangeboden 'as is', zonder enige garantie. De VTM GO naam, VTM GO logo en de kanaallogo's zijn eigendom van DPG Media en worden gebruikt in overeenstemming met de fair use policy.</disclaimer>
         <platform>all</platform>
         <license>GPL-3.0</license>
```

Comment:
please try to fix with a proper spdx identifier in the next release. Options afaik are gpl-3.0-only or gpl-3.0-or-later

Reply:
Alright. Will do!